OpenEdXMobile/src/main/java/org/edx/mobile/view/AuthenticatedWebViewFragment.java

```diff
@@ -22,17 +25,29 @@
  * Javascript can also be passed in arguments for evaluation.
  */
 public class AuthenticatedWebViewFragment extends BaseFragment {
-    protected final Logger logger = new Logger(getClass().getName());
     public static final String ARG_URL = "ARG_URL";
     public static final String ARG_JAVASCRIPT = "ARG_JAVASCRIPT";
     public static final String ARG_IS_MANUALLY_RELOADABLE = "ARG_IS_MANUALLY_RELOADABLE";
+    protected final Logger logger = new Logger(getClass().getName());
 
+    @org.edx.mobile.annotation.Nullable
     @InjectView(R.id.auth_webview)
     protected AuthenticatedWebView authWebView;
 
+    @org.edx.mobile.annotation.Nullable
     @InjectView(R.id.swipe_container)
     protected SwipeRefreshLayout swipeContainer;
 
+    @org.edx.mobile.annotation.Nullable
+    @InjectView(R.id.content_error_text)
+    protected TextView tvContentError;
+
+    @org.edx.mobile.annotation.Nullable
+    @InjectView(R.id.content_error_action)
+    protected Button btnContentErrorAction;
+
+    protected boolean isInflateWebViewLayout = false;
```

Comment:
We can rename it to `isSystemUpdatingWebView`
make a getter of it and put proper java docs over it

Reply:
we are not sure, the crash occurs while updating the `webview`, IMO `isWebViewAvailable` more suitable for this. what do you think?

Reply:
As per team discussion 'isSystemUpdatingWebView' makes more sense.